base/applications/regedit/clb/lang/fr-FR.rc

```diff
@@ -10,7 +10,7 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
 CAPTION "Styles de Boites de la Liste de la Colonne"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-   GROUPBOX "Styles des Colonne de la Liste", -1, 6, 7, 158, 71
+   GROUPBOX "Styles des Colonnes de la Liste", -1, 6, 7, 158, 71
```

Comment:
I think you could fix (here or on a separate PR) the capitalization of words to match what's used in french language.